cloudbuild.yaml

```diff
@@ -0,0 +1,10 @@
+steps:
+- name: golang:1.21
+  entrypoint: /bin/bash
+  args:
+  - -c
+  - |
+    go install go install github.com/google/ko@latest
```

Comment:
```suggestion
    go install github.com/google/ko@latest
```

Double command ?

Reply:
Whoops, thank you for catching that. Updated!